### ISSUE TITLE
Zoom behaviour is different for first channel vs. other channels

### ISSUE THREAD

**Issue by constantinpape** [mobie/spatial-transcriptomics-example-project#21]
The zoom behaviour for sources that come from ome.zarr with multiple channels is different for the first source (behaves correctly), vs. the other channels (behaves incorrectly).

See this video, where everything works correct for `membrane-marker1` (the source corresponding to channel 0), but not for `nuclei` (corresponding to channel 3):

https://user-images.githubusercontent.com/4263537/196243863-65660a79-f2e4-41ce-9da3-754c360b66c4.mp4

@tischi it's probably best to first check if there is an underlying issue in mobie.io via unit-tests. I think I know how to do this (extend the test to the higher scale levels), will try to follow up tomorrow.

**Comment by tischi** [comment]
Is that still an issue?

**Comment by constantinpape** [comment]
Yep, it is. I think I know what's going on, or at least I know how to test for it. I will try to follow up on this.
